hs_core/hydroshare/resource.py

```diff
@@ -263,7 +267,7 @@ def create_resource(
         resource_type, owner, title,
         edit_users=None, view_users=None, edit_groups=None, view_groups=None,
         keywords=(), metadata=None, extra_metadata=None,
-        files=(), fed_res_file_names='', fed_res_path='', fed_copy_or_move=None,
+        files=(), source_names=[], fed_res_path='', move=False,
```

Comment:
@alvacouch According to the docstring, this source_names is a string, so the default should be ```''``` rather than ```[]```